internal/pkg/policy/action.go

```diff
@@ -0,0 +1,62 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package policy
+
+import (
+	"fleet/internal/pkg/model"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/gofrs/uuid"
+)
+
+// Action is a policy action that is sent as an action ID to an agent.
+type Action struct {
```

Comment:
maybe different name? we already have model.Action. could be confusing

Reply:
Change it to `policy.Revision`.